src/Nodes/Structures/SVGStyle.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * This file is part of the Claroline Connect package.
+ *
+ * (c) Claroline Consortium <consortium@claroline.net>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * Date: 2/8/17
+ */
```

Comment:
Sorry about the headers. I've configured my IDE to add this automatically and forgot to delete it.